src/components/base/BaseInput.vue

```diff
@@ -1,35 +1,67 @@
 <script setup lang="ts">
+import { ref, computed } from "vue";
+import EyeIcon from "@/assets/icons/eye.svg?component";
+
 const props = withDefaults(
   defineProps<{
     modelValue?: string;
     inputType?: string;
     inputName?: string;
     placeholder?: string;
+    validation?: string;
```

Comment:
`Validation` may be a bit confusing for what we want to do here since it's pretty general, maybe `errorMessage` would be a better fit. Wdyt

Reply:
Good point and taken - naming is everything.